Source/WebCore/platform/graphics/FontCascade.cpp

```diff
@@ -303,6 +303,12 @@ float FontCascade::widthOfTextRange(const TextRun& run, unsigned from, unsigned
     return offsetAfterRange - offsetBeforeRange;
 }
 
+float FontCascade::width(StringView text, SingleThreadWeakHashSet<const Font>* fallbackFonts, GlyphOverflow* glyphOverflow) const
+{
+    TextRun run { text };
+    return width ( CheckedRef { run }, fallbackFonts, glyphOverflow);
```

Comment:
@rniwa `run` is a stack variable here, there is no way it can get destroyed while running `width()` so the CheckedRef doesn't seem useful here. Either we have a static analysis false positive or the smart pointer adoption here is speculative. If a false positive, we may consider silencing the warning instead.

@vitorroriz : There is an extra space between `width` and `(`.

Reply:
That makes sense, I can check what the static analyzer says when passing a l-value reference. I was looking at [1] when doing this. Let me try a new iteration.

[1] https://build.webkit.org/results/Apple-Sequoia-Safer-CPP-Checks/294967%40main%20%283475%29/scan-build-output/StaticAnalyzer/WebCore/StaticAnalyzerReports/report-FontCascade.h-widthOfSpaceString-1-d3dbd8.html#EndPath